hosted-ce/etc/osg/image-config.d/30-remote-site-setup.sh

```diff
@@ -44,6 +44,21 @@ function fetch_remote_os_info {
     ssh -q "$ruser@$rhost" "cat /etc/os-release"
 }
 
+# Allow the condor user to run the WN client updater as the local users
+function setup_sudo_users {
+    users=$(get_mapped_users)
+    [[ -n $users ]] || errexit "Did not find any HTCondor-CE SCITOKENS user mappings"
+
+    CONDOR_SUDO_FILE=/etc/sudoers.d/10-condor-ssh
+    # Replace spaces and newlines from sort ouput with commas
+    condor_sudo_users=`tr ' \n' ',' <<< $users`
+    # Remove trailing comma from list of sudo-users
+    condor_sudo_users=${condor_sudo_users:0:-1}
```

Comment:
Note: This is where the changes to address the bug actually reside

Reply:
LGTM!